src/utils/v2/math.ts

```diff
@@ -99,7 +99,8 @@ export const redemptionRateFrom = (percentage: string) => {
  * @param feePerBillion - fee as parts-per-billion.
  * @returns {string} fee expressed as a percentage.
  */
-export const formatFee = (feePerBillion: BigNumber) => {
+export const formatFee = (feePerBillion: BigNumber | undefined) => {
+  if (!feePerBillion) return
```

Comment:
I kinda think this pattern throughout the codebase is a bit of a code small. I think e.g. we should conditionally render `ProjectPayoutMods` if `ETHPaymentTerminalFee` is defined, or at least do what I suggested in the above comment.

Let me know what you think

Reply:
Bit on the nose I agree. Will change

Reply:
hahahahaha

Reply:
bit pongy